"Array Iterator" never appear in codes

mfuji09 opened the pull request:
<!-- Please provide the following information to help us review this PR: -->

> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'



> What was wrong/why is this fix needed? (quick summary only)

"Array Iterator" never appear in codes, so I think it shouldn't be styled as a code like `Array Iterator`.

> Anything else that could help us review it

mfuji09 committed:
`Array Iterator` never appear in code

teoli2003 commented on files/en-us/web/javascript/reference/global_objects/array/index.md:
```diff
@@ -327,7 +327,7 @@ The properties and elements returned from this match are as follows:
   - : Joins all elements of an array into a string.
 - {{jsxref("Array.prototype.keys()")}}
-  - : Returns a new `Array Iterator` that contains the keys for each index in the array.
+  - : Returns a new Array Iterator that contains the keys for each index in the array.
```
Let's uniformize this one too.
```suggestion
  - : Returns a new _array iterator_ that contains the keys for each index in the array.
```

teoli2003 commented on files/en-us/web/javascript/reference/global_objects/array/values/index.md:
```diff
@@ -16,7 +16,7 @@ browser-compat: javascript.builtins.Array.values
 
 The **`values()`** method returns a new
-**`Array Iterator`** object that contains the values for each
+**Array Iterator** object that contains the values for each
```
```suggestion
_array iterator_ object that contains the values for each
```

teoli2003 commented on files/en-us/web/javascript/reference/global_objects/bigint64array/index.md:
```diff
@@ -70,7 +70,7 @@ The **`BigInt64Array`** typed array represents an array of 64-bit signed integer
   - : Joins all elements of an array into a string. See also {{jsxref("Array.prototype.join()")}}.
 - {{jsxref("TypedArray.keys", "BigInt64Array.prototype.keys()")}}
-  - : Returns a new `Array Iterator` that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
+  - : Returns a new Array Iterator that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
```
```suggestion
  - : Returns a new _array iterator_ that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
```

teoli2003 reviewed (changes requested):
Let's uniformize a little bit more.

mfuji09 committed:
Update files/en-us/web/javascript/reference/global_objects/array/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

mfuji09 committed:
Update files/en-us/web/javascript/reference/global_objects/array/values/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

mfuji09 committed:
Update files/en-us/web/javascript/reference/global_objects/bigint64array/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

mfuji09 commented:
@teoli2003 I agree!

mfuji09 committed:
fix markup in Web API references

The Web API hasn't been converted in Markdown...

mfuji09 committed:
fix more representations of "array iterator"

I am noticed that there are still "Array Iterator" which should be fixed.
I have fixed them.

mfuji09 commented:
@teoli2003 
And I have found more and I have fixed them too.

teoli2003 commented:
Thanks!
